game_stats.json

```diff
@@ -0,0 +1 @@
+{"player": 2, "computer": 3, "rock": 3, "paper": 8, "scissors": 3}
```

Comment:
Normally you actually wouldn't want a file like this under version control - the values here are going to change whenever someone plays the game. If it's just you, that's going to lead to a lot of unnecessary noise in your commit history and make it hard for you to start over (because any time anything in this file changes, git's going to see it as a change and ask if you want to commit it). So generally what you'd do is list this in a file called `.gitignore` in the root of the directory. It does exactly what it sounds like - tells git to ignore certain files and exclude the from version control. in that file, you'd just put type in the path to this file. Since you've already added this file to version control, you'll also have to delete this file and commit that change along with the `.gitignore` file.